scripts/test.py

```diff
@@ -147,17 +154,17 @@ def make_log_dir():
         print('- %s:' % profile)
         build_log = os.path.join(profile_dir, 'build.log')
         run_cmd(
-            'dune build --display=progress --profile=%s %s 2> %s'
-                % (profile, coda_exe_path, build_log),
+            'dune build --display=progress --profile=%s %s %s 2> %s'
```

Comment:
if `logproc` is already in the `PATH`, can we use that instead of building a copy?